projects/aca-content/src/lib/components/search/search-input/search-input.component.spec.ts

```diff
@@ -59,6 +70,12 @@ describe('SearchInputComponent', () => {
     fixture.destroy();
   });
 
+  it('should collapse the sidenav when the input value is clicked', () => {
```

Comment:
Why title says "input value is clicked"? I don't see you click anything in that test